base/src/main/java/com/mindera/skeletoid/widgets/tooltip/PopupTooltip.kt

```diff
@@ -296,75 +213,101 @@ class PopupTooltip(
         view.getLocationOnScreen(screenPos)
         view.getWindowVisibleDisplayFrame(displayFrame)
 
-        val estimatedTooltipHeight = getValueInPixels(ESTIMATED_HEIGHT_DIPS, context)
-        val tooltipArrowHeight = getValueInPixels(arrowHeight, context).toInt()
-        val tooltipArrowWidth = getValueInPixels(arrowWidth, context)
+        val estimatedTooltipHeight = getValueInPixels(popupHeight, context)
 
-        //Custom tooltip layout
-        val layout = inflater.inflate(layout, null)
+        //Setup the tooltip layout
+        val layout = inflater.inflate(layoutRes, null)
 
-        //Custom text
-        val textView = layout.findViewById<TextView>(textViewId)
+        //Set the tooltip text using the [messageViewResId]
+        val textView = layout.findViewById<TextView>(messageViewResId)
         textView.text = text
 
+        //Setup the elevation for both our text view and our background view
```

Comment:
Can we have a space after `//` ? 🙏 

(same for other comments)

Reply:
yup 👍